hotspot/test/gc/metaspace/CompressedClassSpaceSizeInJmapHeap.java

```diff
@@ -25,6 +25,7 @@
  * @test CompressedClassSpaceSizeInJmapHeap
  * @bug 8004924
  * @summary Checks that jmap -heap contains the flag CompressedClassSpaceSize
+ * @requires vm.bits == 64
```

Comment:
Not sure about this. The flag itself exists also on 32-bit, so the test should work there; it is pointless though since the flag is never used.